src/UI/Actions.hs

```diff
@@ -388,8 +388,22 @@ removeTags ts =
 reloadList :: Action 'BrowseThreads AppState
 reloadList = Action "reload list of threads" applySearch
 
+selectNextUnread :: Action 'BrowseMail AppState
+selectNextUnread =
+  Action { _aDescription = "select next unread"
+         , _aAction = (\s -> let vec = (view (asMailIndex . miListOfMails . L.listElementsL) s)
+                                 cur = (view (asMailIndex . miListOfMails . L.listSelectedL) s) <|> Just 0
```

Comment:
This line can be `cur = vec <|> Just 0`

Reply:
Could it be a mistake? One returns the vector the other view the selected list element ...

Reply:
whoops, I misread.  pls ignore.